dataRetriever/resolvers/baseFullHistoryResolver_test.go

```diff
@@ -0,0 +1,101 @@
+package resolvers
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/multiversx/mx-chain-go/testscommon/storage"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBaseFullHistoryResolver_SearchFirst(t *testing.T) {
+	t.Parallel()
+
+	testKey := []byte("key")
+	testValue := []byte("value")
+	resolver := &baseFullHistoryResolver{
+		storer: &storage.StorerStub{
+			SearchFirstCalled: func(key []byte) ([]byte, error) {
+				assert.Equal(t, testKey, key)
+
+				return testValue, nil
+			},
+		},
+	}
+
+	val, err := resolver.searchFirst(testKey)
+	assert.Nil(t, err)
+	assert.Equal(t, testValue, val)
+}
+
+func TestBaseFullHistoryResolver_GetFromStorage(t *testing.T) {
+	t.Parallel()
+
+	testKey := []byte("key")
+	testValue := []byte("value")
+	testEpoch := uint32(37)
+
+	t.Run("get from epoch returned nil error and not empty buffer", func(t *testing.T) {
+		resolver := &baseFullHistoryResolver{
+			storer: &storage.StorerStub{
+				SearchFirstCalled: func(key []byte) ([]byte, error) {
+					assert.Fail(t, "should have not called SearchFirst")
+
+					return nil, nil
+				},
+				GetFromEpochCalled: func(key []byte, epoch uint32) ([]byte, error) {
+					assert.Equal(t, testKey, key)
+					assert.Equal(t, testEpoch, epoch)
+
+					return testValue, nil
+				},
+			},
+		}
+
+		val, err := resolver.getFromStorage(testKey, testEpoch)
+		assert.Nil(t, err)
+		assert.Equal(t, testValue, val)
+	})
```

Comment:
Formatting / newline needed (and below).

Reply:
I generally prefer inner tests not separated by a new line. We should agree in a future clean code meeting if we will add empty lines or not in this particular case as to slowly align all existing code